src/data/projects.yml

```diff
@@ -156,3 +156,42 @@ categories:
         displayLang: JavaScript
         searchLang: javascript
         stars: 11
+  - title: Native Development
+    url: https://github.com/Esri?q=topic%3Anative-development
+    anchor: native-development
+    color: yellow
+    projects:
+      - title: Indoor Routing Xamarin
```

Comment:
all the other title level showcase projects are user case studies.  this will do for now, but ultimately it would be good to curate something similar for runtime.